Microsoft.Identity.Web/Client/TokenCacheProviders/InMemory/MSALPerUserMemoryTokenCacheProvider.cs

```diff
@@ -122,6 +122,11 @@ private void LoadUserTokenCacheFromMemory()
         /// </summary>
         private void PersistUserTokenCache()
         {
+            string cacheKey = this.GetMsalAccountId();
```

Comment:
What problem is this solving, @kalyankrishna1 ?

Reply:
this broke during testing when I sign out and sign-back in repeatedly 

---
In reply to: [273817454](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/pull/84#discussion_r273817454) [](ancestors = 273817454)